src/Enchantments.cpp

```diff
@@ -167,6 +194,141 @@ bool cEnchantments::IsEmpty(void) const
 
 
 
+unsigned int cEnchantments::GetLevelCap(int a_EnchantmentID)
+{
+	switch (a_EnchantmentID)
+	{
+	case enchProtection:           return 4;
+	case enchFireProtection:       return 4;
+	case enchFeatherFalling:       return 4;
+	case enchBlastProtection:      return 4;
+	case enchProjectileProtection: return 4;
+	case enchRespiration:          return 3;
+	case enchAquaAffinity:         return 1;
+	case enchThorns:               return 3;
+	case enchDepthStrider:         return 3;
+	case enchSharpness:            return 5;
+	case enchSmite:                return 5;
+	case enchBaneOfArthropods:     return 5;
+	case enchKnockback:            return 2;
+	case enchFireAspect:           return 2;
+	case enchLooting:              return 3;
+	case enchEfficiency:           return 5;
+	case enchSilkTouch:            return 1;
+	case enchUnbreaking:           return 3;
+	case enchFortune:              return 3;
+	case enchPower:                return 5;
+	case enchPunch:                return 2;
+	case enchFlame:                return 1;
+	case enchInfinity:             return 1;
+	case enchLuckOfTheSea:         return 3;
+	case enchLure:                 return 3;
```

Comment:
Style: the `case` statements should have one extra indent (indent is dictated by curly-braces).